### PR TITLE
create user function added

### PR DESCRIPTION
**Pull request by AnthonyClarke25**
The db in use the the branch db at the mo so we can use the online console in planetscale to run quires. https://app.planetscale.com/bookshift/bookshift/initial-setup/console. 

Please test by making a new user and seeing if it creates a new one. the console log will be to the server. 

The next step is to modify your get user function to return data about the current user from prisma, but I'm still learning.

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
-import type { Metadata } from 'next';
-import { Inter } from 'next/font/google';
-import '../styles/globals.css';
-import { ClerkProvider } from '@clerk/nextjs';
+import type { Metadata } from "next";
```
Comment (DMCFA) [inline src/app/layout.tsx:1]: please change your linting rules for ' otherwise you'll always have all this linting changes cluttering the files

**Review by DMCFA — APPROVED** [review]
I can see the console.log but nothing on planetscale, which I think it's what I should see at this stage so the user is being created but not on the db yet.

Please change your linting rules too so we don't get it in every PR

File: src/app/layout.tsx (hunk 1 above)
Comment (AnthonyClarke25) [reply src/app/layout.tsx:1]: Sure no prob, just a quick one did you update your environment with the key for the db2? you should be able to https://app.planetscale.com/bookshift/bookshift/initial-setup/console
image

Comment (AnthonyClarke25) [reply src/app/layout.tsx:1]: annoyingly you can only use the console for the branched db

**Pull request merged**